docker-compose.yml

```diff
@@ -1,10 +1,12 @@
 version: '3'
 services:
   decidim:
-    build: '.'
-    working_dir: /code
+    image: decidim/decidim:dev-latest
+    entrypoint: []
+    command: decidim
     volumes:
-      - .:/code
+      - .:/app
+      - bundle:/usr/local/bundle
```

Comment:
Why is this needed? So it can be cached or something? I've used this trick with `node_modules` to prevent them from being overwritten by what's in the host when building the image, but it wasn't a problem for the gem bundle because it does not live inside the mounted folder.

Reply:
Well yeah! This will persist this folder in a docker volume, so data keeps saved across runs. We're using the same image every time we `docker-compose run` anything. We probably didn't notice this before because we were building the image with our exact dependencies. We would have noticed that when changing them, though 💅

Reply:
I see, makes sense!